<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>My Personal Index</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Portfolio: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Matt's</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string> Last Updated: 7/31/2009 </string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Holdings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Active</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ticker</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Closing Price</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Shares</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Avg Price Per Share</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Cost Basis</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gain/Loss</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gain/Loss %</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Account</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Asset Allocation</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Chart</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Performance</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Correlations</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Accounts</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Asset Allocation</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string> Index Start Date: 1/2/2009 </string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mainUpdatePrices"/>
   <addaction name="separator"/>
   <addaction name="mainCompare"/>
   <addaction name="separator"/>
   <addaction name="mainEdit"/>
   <addaction name="mainDelete"/>
   <addaction name="separator"/>
   <addaction name="mainAdd_Portfolio"/>
   <addaction name="separator"/>
   <addaction name="mainOptions"/>
   <addaction name="mainAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mainUpdatePrices">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/refresh.png</normaloff>:/icons/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Update Prices</string>
   </property>
   <property name="toolTip">
    <string>Update Prices</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="mainCompare">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/chart.png</normaloff>:/icons/icons/chart.png</iconset>
   </property>
   <property name="text">
    <string>Compare...</string>
   </property>
   <property name="iconText">
    <string>Compare..</string>
   </property>
   <property name="toolTip">
    <string>Compare...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="mainEdit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Ediy</string>
   </property>
  </action>
  <action name="mainDelete">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete</string>
   </property>
  </action>
  <action name="mainAdd_Portfolio">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Portfolio</string>
   </property>
   <property name="toolTip">
    <string>Add Portfolio</string>
   </property>
  </action>
  <action name="mainOptions">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/options.png</normaloff>:/icons/icons/options.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="toolTip">
    <string>Options</string>
   </property>
  </action>
  <action name="mainAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/about.png</normaloff>:/icons/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
